test/writer.docbook5

```diff
@@ -1320,7 +1320,9 @@ or here: &lt;http://example.com/&gt;
       From <quote>Voyage dans la Lune</quote> by Georges Melies (1902):
     </para>
     <figure>
-      <title>lalune</title>
+      <title>
+        lalune
+      </title>
```

Comment:
I think we generally try to use indentation for block-level content but keep it unindented for inline-level; was there a reason this changed?

Reply:
Oops, my bad. Fixed.